client/src/components/manageProjects/editMeetingTimes.js

```diff
@@ -63,50 +60,23 @@ const EditMeetingTimes = ({
         updatedDate,
       };
 
-      // If the day has been changed, find the next occurence of the changed day
-      if (values.day) {
-        const date = findNextOccuranceOfDay(values.day);
```

Comment:
This portion of the code is what creates a false time stamp. `values` always has a `.day` property in the front end, so the code assumes we are needing tom update the day.

This is the most relevant changes to this PR